xray/plot/plot.py

```diff
@@ -392,7 +393,7 @@ def _plot2d(plotfunc):
         use of a diverging colormap.
     robust : bool, optional
         If True and ``vmin`` or ``vmax`` are absent, the colormap range is
-        computed with robust quantiles instead of the extreme values.
+        computed with 2nd and 98th percentiles instead of the extreme values.
```

Comment:
@jhamman the thought here is to hide complexity from the user by only having them see `robust=True` or `False`. If they want to do anything beyond this they can just do the operation on a `DataArray`.